src/include/planner/plan_util.h

```diff
@@ -44,6 +53,15 @@ class PlanUtil {
   static const std::set<oid_t> GetTablesReferenced(
       const planner::AbstractPlan *plan);
 
+  /**
+  * @brief Get the indexes affected by a given query
+  * @param CatalogCache
+  * @param SQLStatement
+  * @return set of affected index object ids
+  */
+  static const std::set<oid_t> GetAffectedIndexes(
+      catalog::CatalogCache &catalog_cache, parser::SQLStatement *sql_stmt);
```

Comment:
Can `sql_stmt` be null? If not, it's probably a good idea to make it a reference. If the function doesn't modify it, even better to make it `const`.

Reply:
Good point. I have made it a const reference and made appropriate changes required for it.